i18n/events/lightning-talk.i18n.js

```diff
@@ -22,38 +22,38 @@ export default genI18nMessages({
                 table: [
                     {
                         time: '08:00',
-                        content: 'Lightning Talk Registration Opens',
+                        content: 'Lightning Talk registration opens',
                     },
                     {
                         time: '13:00',
-                        content: 'Registration Closes',
+                        content: 'Registration closes',
                     },
                     {
                         time: '13:30',
-                        content: 'Selected Speaker List Announced',
+                        content: 'Selected speaker list announced',
                     },
                     {
                         time: '14:30',
-                        content: 'Selected Speakers Confirmation Deadline',
+                        content: 'Selected speakers confirmation deadline',
                     },
                     {
-                        time: '16:00 - 16:30',
+                        time: '14:50',
                         content:
-                            'Speaker Briefing & Tech Check ' +
+                            'Speaker briefing & tech check ' +
                             '(Outside R0 Main Hall)',
                     },
                     {
                         time: '17:10 (Day 1) / 16:50 (Day 2)',
-                        content: 'Lightning Talk Session',
+                        content: 'Lightning Talk session',
```

Comment:
One small suggestion: could we make the table content Title Case for consistency?
For example:
- Lightning Talk Registration Opens
- Registration Closes
- Selected Speaker List Announced
- Selected Speakers Confirmation
- Deadline Speaker Briefing & Tech Check (Outside R0 Main Hall)
- Lightning Talk Session